scripts/zones/Grand_Palace_of_HuXzoi/mobs/Ix_aern_mnk.lua

```diff
@@ -0,0 +1,82 @@
+-----------------------------------
+-- Area: Grand Palace of HuXzoi
+-- NPC:  Ix_aern_mnk
+-- ID: 16916815
+-----------------------------------
+
+require("scripts/globals/status");
+
+-----------------------------------
+-- onMobInitialize Action
+-----------------------------------
+
+function onMobInitialize(mob)
+end;
+
+-----------------------------------
+-- onMobSpawn Action
+-----------------------------------
+
+function onMobSpawn(mob)
+    local chance = GetServerVariable("[SEA]IxAern_DropRate"); -- Adjust drop rate for the items based on the organs traded.

```

Comment:
o and have you checked those mods apply correctly in onMobInitialize?
i don't remember what it was exactly, but there was something like a mobmod or w/e that couldn't be applied to the mob on initialise and had to be applied on spawn instead

though im probably getting it confused with something else